website/versioned_docs/version-v3.10.x/externaldata.md

```diff
@@ -456,9 +456,9 @@ server := &http.Server{
 
 To ensure a request to the Gatekeeper webhook is coming from the API server, Gatekeeper needs to validate the client cert in the request. To enable authenticate API server, the following configuration can be made:
 
-1. Deploy Gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-cert-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate API server requests if client CA name is provided with flag.
+1. Deploy Gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-ca-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate API server requests if client CA name is provided with flag.
```

Comment:
I don't think you should update the version docs for this change since this change will only land in the next v3.11 release? and pls add a warning to the latest doc to warn users that due to the changes introduced in v3.11 for these 2 flags, because they are not backward compatible, they may need a clean install.